Mathlib/Logic/Equiv/PartialEquiv.lean

```diff
@@ -576,7 +576,9 @@ protected def trans' (e' : PartialEquiv β γ) (h : e.target = e'.source) : Part
   right_inv' y hy := by simp [hy, h]
 
 /-- Composing two partial equivs, by restricting to the maximal domain where their composition
-is well defined. -/
+is well defined.
+Within the `Manifold` namespace, there is the notation `e ≫ f` for this.
```

Comment:
This one seems strange on its face. Can you double check?

Reply:
This notation is defined in `Geometry/Manifold/ChartedSpace.lean`. I'm not sure if this is the best solution; I could also simply add a comment there.